pkg/apps/application.jsx

```diff
@@ -100,7 +100,7 @@ export class Application extends React.Component {
         function render_comp() {
             if (!comp) {
                 if (metainfo_db.ready)
-                    return <div>{_("Unknown Application")}</div>;
+                    return <div>{_("Unknown application")}</div>;
```

Comment:
So this "Change strings from sentance case to title case" commit was done entirely automatically with the script? Or does it still have some manual adjustments? The commit message should point out exactly how it was done, so that it can be reproduced.

Also, it would be really good to adjust the msgids in *.po as well in the same commit.

Reply:
> So this "Change strings from sentance case to title case" commit was done entirely automatically with the script?

yes. Well I did `make title2senance` and then just ran `t2s.sh` and commit what it changed. But no manual intervention.

> Also, it would be really good to adjust the msgids in *.po as well in the same commit.

I guess I could do that (with un-wrapping it first). I thought I would do it just in the weblate repo and then sync it back here. But if this is acked I can just run that `t2s.sh` script on `po/` no problem with that.